apps/web/src/pages/templates/workflow/workflow/node-types/WorkflowNode.tsx

```diff
@@ -367,22 +423,31 @@ const WorkflowNodeWrapper = styled.div`
   width: 100%;
 `;
 
-const ErrorCircle = styled.div<{ dark: boolean }>`
+const ErrorCircle = styled.div<{ dark: boolean; alignment: 'left' | 'right' }>`
   width: 11px;
   height: 11px;
   display: inline-block;
   position: absolute;
-  right: -6px;
+  ${({ alignment }) => alignment}: -6px;
   top: calc(50% - 4px);
   background: ${colors.error};
   border-radius: 50%;
   border: 3px solid ${({ dark }) => (dark ? colors.B15 : 'white')};
 `;
 
-const ActionWrapper = styled.div`
+const ActionWrapper = styled.div<{ showGroupError?: boolean }>`
   display: flex;
-  height: 28px;
-  margin: 4px 0 4px 4px;
+  align-items: center;
+  height: 40px;
+  padding: 4px 12px;
+  border-top-right-radius: 7px;
+  border-top-left-radius: 7px;
+  ${({ showGroupError }) =>
+    showGroupError
+      ? `
+      background: linear-gradient(0deg, rgba(229, 69, 69, 0.20) 0%, rgba(229, 69, 69, 0.20) 100%), #23232B;
```

Comment:
Nit: this gradient likely belongs in the design system. Do we have other places that share this? I'm mostly interested in how we arrived at with the `rgba` and hex values, and if we can reuse existing values to compose the gradient.